components/homme/src/share/cxx/PpmRemap.hpp

```diff
@@ -178,16 +184,21 @@ template <typename boundaries> struct PpmVertRemap : public VertRemapAlg {
   const int gs = _ppm_consts::gs;
 
   explicit PpmVertRemap(const int num_elems, const int num_remap)
-      : dpo("dpo", num_elems), pio("pio", num_elems), pin("pin", num_elems),
-        ppmdx("ppmdx", num_elems), z2("z2", num_elems), kid("kid", num_elems),
-        ao("a0", get_num_concurrent_teams<ExecSpace>(num_elems * num_remap)),
-        mass_o("mass_o",
-               get_num_concurrent_teams<ExecSpace>(num_elems * num_remap)),
-        dma("dma", get_num_concurrent_teams<ExecSpace>(num_elems * num_remap)),
-        ai("ai", get_num_concurrent_teams<ExecSpace>(num_elems * num_remap)),
-        parabola_coeffs(
-            "Coefficients for the interpolating parabola",
-            get_num_concurrent_teams<ExecSpace>(num_elems * num_remap)) {}
+      : m_dpo("dpo", num_elems)
+      , m_pio("pio", num_elems)
+      , m_pin("pin", num_elems)
+      , m_ppmdx("ppmdx", num_elems)
+      , m_z2("z2", num_elems)
+      , m_kid("kid", num_elems)
+      , m_ao("a0", get_num_concurrent_teams<ExecSpace>(num_elems * num_remap))
+      , m_mass_o("mass_o",get_num_concurrent_teams<ExecSpace>(num_elems * num_remap))
+      , m_dma("dma", get_num_concurrent_teams<ExecSpace>(num_elems * num_remap))
+      , m_ai("ai", get_num_concurrent_teams<ExecSpace>(num_elems * num_remap))
+      , m_parabola_coeffs("Coefficients for the interpolating parabola",
+            get_num_concurrent_teams<ExecSpace>(num_elems * num_remap))
+  {
+    // Nothing to do here
+  }
```

Comment:
These style/indenting changes should go in a separate PR.

Reply:
Why is it a problem to have them with this PR? I thought formatting PR were not even a thing in E3SM. Here, I changed the layout cause it was faster (in vim) to do the change and prepend the `m_`, rather than just pre-pending it. Granted, I also think it's way more readable, but if we want to stick with the no-style changes, I can put them back in the order they were before.

Reply:
Because if you're looking at diffs to understand how the PR fulfilled its purpose, all this other stuff shows up.  You can do a formatting-only PR if you want.

Reply:
Well, this also changed the names of the member variables (to avoid variable shadowing warnings in some functions), so I guess they would show up anyways...

Reply:
ok.